api-tests/features/p2-m2b.feature

```diff
@@ -0,0 +1,34 @@
+@Build
+Feature: M2B No Photo Id Journey
+
+  Scenario: M2B Journey
+    Given I start a new 'medium-confidence' journey with feature set 'm2bBetaExperianKbv'
```

Comment:
I was sort of expecting these to be two steps, but this is fine too.

In general the feature set would need to come first (although for most features it won't matter), so it'd be:
```
Given I have the feature set 'm2bBetaExperianKbv' enabled
And I start a new 'medium-confidence' journey
...
```